CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `getNFTokenID` to get the NFTokenID after minting a token.
 - Added `LedgerEntryType` enum and added `type` field to `Ledger` and `LedgerData` requests
 - Added the algorithm used to encode a wallet's seed to the wallet.
+- Added `submit_and_wait` to sign (if needed), autofill, submit a transaction and wait for its final outcome
```

Comment:
```suggestion
- Added `submit_and_wait` to autofill, sign, and submit a transaction and wait for its final outcome
```

Reply:
The changelog should also have a note about `send_reliable_submission` now accepting a `fail_hard` param

Reply:
Also, this needs to move to the `Unreleased` section